j2lint/logger.py

```diff
@@ -3,17 +3,22 @@
 import logging
 from logging.handlers import RotatingFileHandler
 from logging import handlers
-
+import sys
 JINJA2_LOG_FILE = "jinja2-linter.log"
 
 logger = logging.getLogger('')
-logger.setLevel(logging.INFO)
-
 
-def add_handler(logger):
+def add_handler(logger, stream='', stream_level=''):
     log_format = logging.Formatter(
         "%(asctime)s - %(name)s - %(levelname)s - %(message)s")
-    fh = handlers.RotatingFileHandler(
-        JINJA2_LOG_FILE, maxBytes=(1048576*5), backupCount=4)
-    fh.setFormatter(log_format)
-    logger.addHandler(fh)
+    if not stream:
```

Comment:
logger.setLevel(log_level) should be added before if else condition since it is separate from the handler. Then it wont be needed in if else condition